app/views/metrics/index.html.erb

```diff
@@ -48,8 +48,14 @@
           </div>
 
           <div class="m-search-filter hidden" data-behaviour="m-search-filter">
-            <label for="search-departments" class="visuallyhidden">Find department</label>
-            <input type="search" id="search-departments" class="a-search-input" placeholder="Find department">
+            <label for="search-metrics">
+              Find
+              <% if @metrics.group_by == 'department' %> department
```

Comment:
I'd prefer not use "magic" strings here. I think we can use the constants in `Metrics::Group` here. Also you might want to consider defining a hash to query here, mapping from the selected group to the string to use. Then the code might look something like this:

```
<%= SearchFilterGroupName.fetch(@metrics.group_by, " by name") %>
```